components/financial_assistance/app/views/financial_assistance/shared/_fa_left_nav.html.erb

```diff
@@ -13,7 +13,7 @@
 
         <li class="list-box step-tabs <%= li_nav_classes_for(:tax_info) %> applicant-nav">
           <% if @applicant.tax_info_complete?%>
-            <span class="fa fa-check applicant-check"></span>
+            <span class="fa fa-check applicant-check"><span class="hide"><%=l10n(".completed")%></span></span>
```

Comment:
I could be wrong, but i don't think the l10n is actually referencing the value in the translation file as is. I think the . is incorrect syntax? 

We have some processing on l10n fields that, as a fallback if it cant find it in the translation file, will take whatever is in the quotes and just add spaces and correct case. I think is actually why this is probably rendering correctly without being actually linked.

Reply:
After some discussion off the ticket and doing some testing with it, `l10n("completed")` is the proper format (unless it is stored in the translations file as `"en.insured.completed"`, which would need us to use `l10n("insured.completed")`). I have updated this ticket and tried to fix places I have used the incorrect format previously: https://github.com/ideacrew/enroll/pull/3431/commits/c205c244b2dd738a90f1e9fa653c7218e315adb0